KeystoneHQ/keystone3-firmware
# Build: split build config

## Explanation
Supports compiling multi-coins and btc-only firmware separately

## Pre-merge check list

## How to test
Compile multi-coins and btc-only firmware respectively, and then verify each function.
For details on compilation method, see README file.
